govuk_publishing_components.gemspec

```diff
@@ -24,6 +24,7 @@ Gem::Specification.new do |s|
   s.add_dependency "rouge"
   s.add_dependency "sprockets", ">= 3"
 
+  s.add_development_dependency "awesome_print"
```

Comment:
@DilwoarH This dependency is used outside development, as component guide is also used in apps that run this gem (whereas development dependencies are only installed when you're debugging the gem).

This is why Static had problems with this - and any other apps would (unless they already had the `awesome_print` gem installed. 

Rather than moving this to be an app dependency (and adding the overhead that all the apps then need to manage it). I'd consider

a) Do we need components to be fussy about whether things are a symbol or a string? it's quite precise - but maybe the horse has bolted
b) Lets see if we can use `pp` instead which is part of Ruby Standard Library, so wouldn't need managing by applications (which seems a lot for one line of code)

Reply:
Thanks for this @kevindew, this has now been reverted.

Regarding your points.

a) I think this is probably the best way to move forward. The change to the documentation was to display the documentation as expected by the author.
b) This is the one I initially went with but it is very limited and styling standards didn't meet the expected coding styles we had in place already.

I will speak with the rest of the community and look at the solutions suggested.